jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SamInitSelectionPanel.java

```diff
@@ -39,7 +39,7 @@
 import software.aws.toolkits.jetbrains.services.lambda.RuntimeGroupUtil;
 import software.aws.toolkits.jetbrains.services.lambda.sam.SamExecutable;
 
-public class SamInitSelectionPanel implements ValidatablePanel {
```

Comment:
What did `ValidatablePanel` do to begin with?

Reply:
Nothing, theres no reference to it anywhere but here....

This should have gone in #2112 but got into wrong change set